File: Data Structures & Algorithms/majority-element-ii/submission-0.go
func majorityElement(nums []int) []int {
	n := len(nums) / 3
	result := make([]int,0,len(nums))
	seenMap := make(map[int]int)
	for _, val := range nums {
		seenMap[val]++
	}

	for key,value := range seenMap {
		if value > n {
			result = append(result,key)
		}
	}
	return result
}

File: Data Structures & Algorithms/set-mismatch/submission-0.py
class Solution:
    def findErrorNums(self, nums: List[int]) -> List[int]:
        freq = [0] * (len(nums) + 1)
        result = [0] * 2

        for num in nums:
            freq[num] += 1

        for i in range(len(freq)):
            if freq[i] == 2:
                result[0] = i

            if freq[i] == 0:
                result[1] = i

        return result


